client/platform/web-girder/views/ViewerLoader.vue

```diff
@@ -84,8 +95,9 @@ export default defineComponent({
 <template>
   <Viewer
     :id="id"
-    :key="id"
+    :key="id + revisionNum"
```

Comment:
For safety, would it be better to include a delimiting character like `/` between `id` and `revisionNum`?